spec/acceptance/nodesets/default.yml

```
@@ -56,6 +56,9 @@ CONFIG:
   log_level: verbose
   type:      aio
   vagrant_memsize: 256
+  ssh:
+    keepalive: true
+    keepalive_interval: 1
```

I *think* this should work

```suggestion
  ssh:
    keepalive: true
    keepalive_interval: 1
    kex:
      - ecdh-sha2-nistp521
      - ecdh-sha2-nistp384
      - ecdh-sha2-nistp256
      - diffie-hellman-group-exchange-sha256
      - diffie-hellman-group14-sha1
      - diffie-hellman-group-exchange-sha1
      - diffie-hellman-group1-sha1
    encryption:
      - aes256-ctr
      - aes192-ctr
      - aes128-ctr
      - aes256-cbc
      - aes192-cbc
      - aes128-cbc
    hmac:
      - hmac-sha2-512
      - hmac-sha2-256
      - hmac-sha1
```